packages/manager/apps/hub/src/dashboard/incident/migration/contracts/template.html

```diff
@@ -0,0 +1,25 @@
+<form name="contractsToAgree" novalidate>
+    <oui-modal
+        heading="{{:: 'manager_hub_incident_migration_contract' | translate}}"
```

Comment:
```suggestion
        heading="{{:: 'manager_hub_incident_migration_contract' | translate }}"
```

Reply:
Non-blocking, should have linting for that if we really want to format AngularJS mustache expression in HTML file.